pyproject.toml

```diff
@@ -2,5 +2,62 @@
 requires = ["setuptools", "setuptools-scm"]
```

Comment:
Unrelated to this commit, but I think you can remove "setuptools-scm" from here, as it's not actually use (the version is statically defined in `__init__.py`).